centrifuge-app/src/components/IssuerSection.tsx

```diff
@@ -2,46 +2,78 @@ import { PoolMetadata } from '@centrifuge/centrifuge-js'
 import { useCentrifuge } from '@centrifuge/centrifuge-react'
 import { Accordion, AnchorButton, Box, IconExternalLink, Shelf, Text } from '@centrifuge/fabric'
 import * as React from 'react'
+import { useLocation } from 'react-router'
 import { ExecutiveSummaryDialog } from './Dialogs/ExecutiveSummaryDialog'
 import { LabelValueStack } from './LabelValueStack'
 import { PillButton } from './PillButton'
-import { AnchorTextLink } from './TextLink'
+import { AnchorTextLink, RouterTextLink } from './TextLink'
 
 type IssuerSectionProps = {
   metadata: Partial<PoolMetadata> | undefined
 }
 
+const reportLinks = [
+  { label: 'Balance sheet', href: '/balance-sheet' },
+  { label: 'Profit & loss', href: '/profit-and-loss' },
+  { label: 'Cashflow statement', href: '/cash-flow-statement' },
+  { label: 'View all', href: '/' },
+]
+
 export function ReportDetails({ metadata }: IssuerSectionProps) {
   const cent = useCentrifuge()
+  const { pathname } = useLocation()
   const report = metadata?.pool?.reports?.[0]
+
   return (
-    report && (
-      <>
-        <Shelf gap={1}>
-          {report.author.avatar?.uri && (
-            <Box
-              as="img"
-              height={40}
-              borderRadius={30}
-              src={cent.metadata.parseMetadataUrl(report.author.avatar.uri)}
-              alt=""
-            />
-          )}
-          <Text variant="body2">
-            Reviewer: {report.author.name}
-            <br />
-            {report.author.title}
+    <Shelf flexDirection="column" alignItems="flex-start">
+      <Shelf marginBottom={20}>
```

Comment:
Also would make this 
```suggestion
      <Shelf marginBottom={30}>
```